src/metadataGeneration/typeResolver.ts

```diff
@@ -323,20 +324,18 @@ export class TypeResolver {
       const inheritedProperties = this.getModelInheritedProperties(modelType) || [];
       const example = this.getNodeExample(modelType);
 
-      const referenceType = {
+      const referenceType: Tsoa.ReferenceType = {
         additionalProperties,
         dataType: 'refObject',
         description: this.getNodeDescription(modelType),
+        example,
```

Comment:
@dgreene1 I put it here. Oddly enough I changed it here instead of just adding a note asking what's the difference between checking for `undefined` (or technically falsy values) before assigning. Must've been late when I did this.